apps/fulfillment-e2e/src/integration/start-picking-picklist-with-customer-note.cy.ts

```diff
@@ -73,4 +73,52 @@ describe('Start picking a picklist with customer note', () => {
       pickingFragment.getProducts().should('have.length.at.least', 1);
     });
   });
+
+  describe('and picking is already in progress', () => {
```

Comment:
same wait and small `it` tests that should be combined here :)

Reply:
I still ended up having to keep the wait because it didn't work without it. It's using the same format as the `beforeEach` above it: https://github.com/spryker/oryx/blob/488ca865257b8e92c8ef95331a24519719bb152a/apps/fulfillment-e2e/src/integration/start-picking-picklist-with-customer-note.cy.ts#L47-L48